metashare/api/models.py

```diff
@@ -502,6 +513,29 @@ def add_commits(self, commits, sender):
     def add_ms_git_sha(self, sha):
         self.ms_commits.append(sha)
 
+    def queue_submit_review(self, *, user, data):
+        from .jobs import submit_review_job
+
+        self.currently_submitting_review = True
+        self.save()
+        self.notify_changed()
+        submit_review_job.delay(user=user, task=self, data=data)
+
+    def finalize_submit_review(
+        self, timestamp, err=None, status=None, delete_org=False
+    ):
+        self.review_submitted_at = timestamp
+        self.currently_submitting_review = False
+        if err:
+            self.review_valid = False
```

Comment:
I'm not sure why we mark the review as invalid if there's an error. `review_valid: False` indicates that the review is out-of-date -- if there's an error I don't think we should be changing the value at all.

Reply:
Addressed in https://github.com/oddbird/MetaShare/pull/164/commits/a758fc098e144db867c390ad4926f7bdecd667ab